docs/content/kitty-items/get-started.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Getting started
+description: Clone, install, and start the project
+sidebar_title: Getting started
+---
+
+The following steps will guide you through the steps required to spin up your own version of Kitty Items.
+
+## 🛠 Check pre-requisites
+
+Before moving to the next step, ensure you have the following tools installed:
+
+### Docker
+
+Ensure you have Docker [installed](https://www.docker.com/get-started) and running. Version **1.13.0+** is required.
+
+```
+> docker version
+
+[...]
+Server: Docker Engine - Community
+ Engine:
+  Version:          20.10.12
+[...]
+```
```

Comment:
This is going to go away soon. 
https://github.com/onflow/kitty-items/issues/214